app/models/user.rb

```diff
@@ -6,9 +6,7 @@ class User < ApplicationRecord
   validates :email, email: { strict_mode: true }, if: "uid.nil?", on: :create
 
   enum role: %w(customer venue_admin platform_admin)
-
-  after_create :send_welcome_email
```

Comment:
@jmejia The method does still exist in the User model. Would it be more appropriate to remove it and just call the code in the controller?